result/brace-style.js

```diff
@@ -0,0 +1,209 @@
+if (foo) {
+  bar();
+} else {
+  baz();
+}
+if (foo) {
+  bar();
+}
+else {
+  baz();
+}
+if (foo)
+{
+  bar();
+}
+else
+{
+  baz();
+}
+/*eslint brace-style: "error"*/
+
+function foo()
+{
+  return true;
+}
+
+if (foo)
+{
+  bar();
+}
+
+try
+{
+  somethingRisky();
+} catch(e)
+{
+  handleError();
+}
+
+if (foo) {
+  bar();
+}
+else {
+  baz();
+}
+/*eslint brace-style: "error"*/
+
+function foo() {
+  return true;
+}
+
+if (foo) {
+  bar();
+}
+
+if (foo) {
+  bar();
+} else {
+  baz();
+}
+
+try {
+  somethingRisky();
+} catch(e) {
+  handleError();
+}
+
+// when there are no braces, there are no problems
+if (foo) bar();
+else if (baz) boom();
+/*eslint brace-style: ["error", "1tbs", { "allowSingleLine": true }]*/
+
+function nop() { return; }
+
+if (foo) { bar(); }
+
+if (foo) { bar(); } else { baz(); }
+
+try { somethingRisky(); } catch(e) { handleError(); }
+/*eslint brace-style: ["error", "stroustrup"]*/
+
+function foo()
+{
+  return true;
+}
+
+if (foo)
+{
+  bar();
+}
+
+try
+{
+  somethingRisky();
+} catch(e)
+{
+  handleError();
+}
+
+if (foo) {
+  bar();
+} else {
+  baz();
+}
+/*eslint brace-style: ["error", "stroustrup"]*/
+
+function foo() {
+  return true;
+}
+
+if (foo) {
+  bar();
+}
+
+if (foo) {
```

Comment:
__[development][eslint]__
* Error - Opening curly brace appears on the same line as controlling statement. (brace-style)